config/newrelic.yml

```diff
@@ -1,6 +1,5 @@
 common: &default_settings
   license_key: <%= ENV['NEW_RELIC_LICENSE_KEY'] %>
-  app_name: <%= "#{ENV['NEW_RELIC_APP_NAME']}-#{ENV['RAILS_ENV']}" %>
```

Comment:
This should work in all environments except maybe development... I'd consider just leaving this and changing development only if that's the case.